fm_banks/wopn2hpp.sh

```diff
@@ -1,25 +1,34 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Why `bash` and not `sh`? Mainly because `sh` appears too ancient and lacks many things which making script be not working or very hard to manage. However, this case is dependent on a platform: on first range of platforms the sh becomes a copy of bash, on others, the real sh appears, which makes a lot of issues. Therefore I suggesting to keep the `bash` to avoid any possible issues with ancient `sh`.

Reply:
If problems arise, I will be very surprised.

Reply:
Look: on CentOS the `/bin/sh` is a symlink into bash. Here the script will work fine.
HOWEVER:
on Linux Mint/Debian/Ubuntu/etc. the /bin/sh is a symlink into `dash` thing which is VERY simplified and limited
![Снимок экрана от 2019-12-04 16-37-01](https://user-images.githubusercontent.com/6751442/70146971-6e610d80-16b4-11ea-86d8-13819666d55a.png)
I don't know what happens on other distros and on xBSD, however, I guess, on xBSD is something worse or same appears as `sh` in my memory... Gonna to make a check on my end...

Reply:
Anyway, one of exapmles that makes pain on `sh` is lack of `read -n 1` which I use to simulate "pause" command for some scripts.

Reply:
Debian Wheezy (sh -> /bin/dash). Works.

Reply:
I did a little test on my end, yeah, it works, however, I prefer to use `/bin/bash` for most of things. Anyway, as this script is simple and doesn't requires more complex logic, it's fine to use `/bin/sh` here.

Reply:
`read -n 1` -> `sleep 1`

Reply:
WARNING! I forgot to replace `-` with `_` when forming the variable name!

Reply:
Fixed: https://github.com/Wohlstand/libOPNMIDI/pull/96/commits/8e18b6eec482bc919124a60fb22d2fe9415ecc57